static/switch-btn.js

```diff
@@ -39,7 +39,7 @@ function switchRight(){//右邊，10連
 	activeSwitch.style.left = '50%';
 	numDraws.value=10;
 	changeMachine('#FF0000', '#FF4500', '#B22222', '#8B0000');
-	achine('#FFD700', '#FFA500', '#FF6347', '#d76a1d');
+	achine('#FFD700', '#FFA500', '#FF6347', '#d76a1d');// Suspected Redundant Code
```

Comment:
what is this?

Reply:
Time to git blame.png

Reply:
@iach526526 

https://github.com/SCAICT/SCAICT-uwu/commit/6606febd42daa7a25f4cc9f279c13a49e65dd25b

Reply:
(test: this should not trigger Merge CI again.)

Reply:
(the only possible problem)

Reply:
Which color scheme you want to keep ><
![image psd](https://github.com/user-attachments/assets/ee7f82e2-1800-4057-a3e1-903f74b901f8)

Reply:
I asked iach and he said that the yellow one is original which should be deprecated. as for 50 times...maybe wait until needed(?), not have a clear replies . is now the time to delete the line and mark this as resolved?  or do you have other plans? @winstonsung ?

Reply:
Yeah, let's just remove the yellow one.